packages/deployments/contracts/contracts/Connext.sol

```diff
@@ -679,6 +701,11 @@ contract Connext is
       return;
     }
 
+    // If the sender is not approved relayer, revert()
+    if (!approvedRelayers[msg.sender]) {
+      revert Connext__handleRelayerFees_notApprovedRelayer();
+    }
```

Comment:
@rhlsthrm we need the address of the gelato exec contract to make sure we don't break testnet